Example/Tests/GINIMultipageReviewControllerTests.swift

```diff
@@ -0,0 +1,187 @@
+//
+//  GINIMultipageReviewControllerTests.swift
+//  GiniVision_Tests
+//
+//  Created by Enrique del Pozo Gómez on 1/30/18.
+//  Copyright © 2018 Gini GmbH. All rights reserved.
+//
+
+import XCTest
+@testable import GiniVision
+
+final class GINIMultipageReviewControllerTests: XCTestCase {
+    
+    lazy var vc: MultipageReviewController = {
+        let vc = MultipageReviewController(imageDocuments: self.imageDocuments)
+        _ = vc.view
+        return vc
+    }()
+    
+    lazy var imageDocuments: [GiniImageDocument] = [
+        self.loadImageDocument(withName: "invoice"),
+        self.loadImageDocument(withName: "invoice2"),
+        self.loadImageDocument(withName: "invoice3")
+    ]
+    
+    func testCollectionsItemsCount() {
+        XCTAssertEqual(vc.mainCollection.numberOfItems(inSection: 0), 3,
+                       "main collection items count should be 3")
+        
+        XCTAssertEqual(vc.bottomCollection.numberOfItems(inSection: 0), 3,
+                       "bottom collection items count should be 3")
+    }
+    
+    func testMainCollectionCellContent() {
+        let firstCell = vc.collectionView(vc.mainCollection,
+                                          cellForItemAt: IndexPath(row: 0, section: 0)) as? MultipageReviewMainCollectionCell
+        let secondCell = vc.collectionView(vc.mainCollection,
+                                           cellForItemAt: IndexPath(row: 1, section: 0)) as? MultipageReviewMainCollectionCell
+        let thirdCell = vc.collectionView(vc.mainCollection,
+                                          cellForItemAt: IndexPath(row: 2, section: 0)) as? MultipageReviewMainCollectionCell
+        XCTAssertEqual(firstCell?.documentImage.image, imageDocuments[0].previewImage,
+                       "First cell image should match the one passed in the initializer")
+        XCTAssertEqual(secondCell?.documentImage.image, imageDocuments[1].previewImage,
+                       "Second cell image should match the one passed in the initializer")
+        XCTAssertEqual(thirdCell?.documentImage.image, imageDocuments[2].previewImage,
+                       "Third cell image should match the one passed in the initializer")
+        XCTAssertEqual(firstCell?.documentImage.contentMode, UIViewContentMode.scaleAspectFit,
+                       "First cell content mode should match the one passed in the initializer")
+    }
+    
+    func testMainCollectionCellSize() {
+        let vc = MultipageReviewController(imageDocuments: imageDocuments)
```

Comment:
Why don't you use the view controller you already have initialized?

Reply:
I forget to remove this when I created the separated vc. 👍